runtime/near-vm-runner/src/logic/tests/bls12381.rs

```diff
@@ -39,6 +39,28 @@ mod tests {
         }};
     }
 
+    /// Precompile input and output data struct
+    #[derive(Debug, Clone, serde::Deserialize)]
+    pub struct PrecompileStandaloneData {
+        pub input: String,
+        pub output: String,
+    }
+
+    /// Standalone data for precompile tests.
+    /// It contains input data for precompile and expected
+    /// output after precompile execution.
+    #[derive(Debug, Clone, serde::Deserialize)]
+    pub struct PrecompileStandalone {
+        pub precompile_data: Vec<PrecompileStandaloneData>,
+    }
+
+    impl PrecompileStandalone {
+        fn new(path: &str) -> Self {
+            let data = fs::read_to_string(path).expect("Unable to read file");
```

Comment:
I'd say lets `include_str!()` the json file instead of loading it at runtime. I wouldn't want to bet on any particular working directory for the tests being run.